physics/Interstitials/UFS_SCM_NEPTUNE/GFS_debug.F90

```diff
@@ -385,7 +385,7 @@ subroutine GFS_diagtoscreen_run (Model, Statein, Stateout, Sfcprop, Coupling,
                                        nthreads, blkno, errmsg, errflg)
 
 #ifdef MPI
```

Comment:
The CCPP framework developers decided to remove the remaining `ifdef MPI` and associated build system logic to pass `-DMPI` in a follow-up PR, since these changes would make your PR even larger.